k2/csrc/compose.cu

```diff
@@ -9,11 +9,11 @@
  * See LICENSE for clarification regarding multiple authors
  */
 
-#include "k2/csrc/fsa_algo.h"
-
 #include <limits>
 #include <vector>
 
+#include "k2/csrc/fsa_algo.h"
```

Comment:
there are a lotof changes to this file, were these manual?  were they a reversion of something Haowen did?